source/Cosmos.System2/Network/TempDictionary.cs

```diff
@@ -7,9 +14,9 @@ namespace Cosmos.System.Network
     /// TempDictionary template class.
     /// </summary>
     /// <typeparam name="TValue">TempDictionary type name.</typeparam>
-    internal class TempDictionary<TValue>
+    internal class TempDictionary<TKey, TValue>
```

Comment:
Maybe I should have just written my question once here. Why can we not just use the standard Dictionary?